packages/disputer/test/Disputer.js

```diff
@@ -90,18 +92,25 @@ contract("Disputer.js", function(accounts) {
         await collateralToken.mint(sponsor3, convert("100000"), { from: contractCreator });
         await collateralToken.mint(liquidator, convert("100000"), { from: contractCreator });
         await collateralToken.mint(disputeBot, convert("100000"), { from: contractCreator });
+
+        // Create identifier whitelist and register the price tracking ticker with it.
+        identifierWhitelist = await IdentifierWhitelist.new();
```

Comment:
This basically shows the main changes that the tests required: removing all `.deployed` syntax and replacing it with actual deployed contract instances for each instance.

Reply:
looks like for the most part the api is the same, you just have to instantiate with the new function?

Reply:
yup. using the `new` keyword is deploying the contract here rather than relying on the migrations with the `.deployed` keyword.